src/payment/bigpay-response.ts

```diff
@@ -0,0 +1,6 @@
+export default interface BigpayResponse<T = any> {
```

Comment:
Can we name it as `PaymentResponse` instead of using an internal name i.e.: `BigPay`?